examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
Ill just remove the flag modifications, I was worried that the MySQL team may have chosen to use /MT for a reason (performance, reliability, etc...). But it seems that is not the case as the default for the DB and the C++ Connector is either (version 5.55 and above). I am guessing the C Connector has just been left behind.